community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup.py

```diff
@@ -224,6 +224,25 @@ def setup_sudoers():
     sudoers_file.chmod(0o0440)
 
 
+def setup_maintenance_script():
+    perform_maintenance = """#!/bin/bash
+
+#SBATCH --priority=low
+#SBATCH --time=180
+
+## Not really, may be sudo reboot ???
+gcloud alpha compute instances perform-maintenance $VM_NAME \
```

Comment:
TODO: Verify command

Reply:
Where do `PROJECT_ID` and `ZONE` come from?

Reply:
metadata